src/BinaryBuilder.jl

```diff
@@ -2,13 +2,11 @@
 # __precompile__()
 
 module BinaryBuilder
-using Compat
+using Compat, Compat.Libdl, Compat.LibGit2, Compat.Random
 using Reexport
 using ObjectFile
+using Nullables
```

Comment:
I would like to avoid having to depend on `Nullables`; is that necessary?  I don't see any `Nullable` stuff here; `Compat` should be providing `Nothing` already.

Reply:
yeah, there are two uses of Nullable in src/wizard/deploy.jl; I'll see if I can get rid of them

Reply:
Hmmmm, looks like 0.6 LibGit2 requires the Nullables for `push`. What do you want to do?

Reply:
Curses.  Let's include `Nullables` then.  Thanks so much for digging through this.  As a reward for all your hard work, I am about to open a new PR that makes our `build_tarballs.jl` significantly simpler.

Reply:
np. Luckily 0.7 is imminent and then we can all get rid of all this compat junk making our code ugly.